src/shared/components/community/community-form.tsx

```diff
@@ -268,9 +268,24 @@ export class CommunityForm extends Component<
               onChange={linkEvent(this, this.handleCommunityVisibilityChange)}
               value={this.state.form.visibilty ?? "Public"}
             >
-              <option value="Public">{I18NextService.i18n.t("public")}</option>
-              <option value="LocalOnly">
-                {I18NextService.i18n.t("local_only")}
+              <option value="Public">
+                {I18NextService.i18n.t("community_visibility_public")}
+              </option>
+              <option value="Unlisted">
+                {I18NextService.i18n.t("community_visibility_unlisted")}
+              </option>
+              <option value="LocalOnlyPublic">
```

Comment:
Check the case on all these, because it'll be unchecked. Or maybe we'll handle that in another issue.